client/project.go

```diff
@@ -49,8 +49,15 @@ func (client *ApiClient) ProjectCreate(payload ProjectCreatePayload) (Project, e
 		return Project{}, err
 	}
 
-	request := map[string]interface{}{"name": payload.Name, "organizationId": organizationId, "description": payload.Description}
-	err = client.http.Post("/projects", request, &result)
+	payloadWith := struct {
```

Comment:
Maybe we should have a better name for it? `payloadWithOrganizationId`, or something of the sort?